app/controllers/api/v2/job_templates_controller.rb

```diff
@@ -27,6 +27,7 @@ def index
       param :overwrite, :bool, :required => false, :desc => N_('Overwrite template if it already exists')
       def import
         options = params[:overwrite] ? { :update => true } : { :build_new => true }
+        options[:associate] = 'new'
```

Comment:
Now that I've had time to look at what we do for other template types, I'd say it would be better to stay consistent with what we do there - adding the template_import_options param group here and letting the user control how it behaves rather than going our own way here.

Reply:
Applied and updated description.